src/mscorlib/src/System/TimeZoneInfo.cs

```diff
@@ -327,7 +327,7 @@ private class OffsetAndRule {
         //
         public AdjustmentRule [] GetAdjustmentRules() {
             if (m_adjustmentRules == null) {
-                return new AdjustmentRule[0];
+                return Array.Empty<AdjustmentRule>();
```

Comment:
This isn't currently available in .NET Core.  Since it's exposed publically and thus has non-zero compat risk, let's revert this one, too.  If in the future this method is exposed, it might be worthwhile revisiting this one, and potentially returning a cached array for FEATURE_CORECLR.